Added activate/deactivate scripts

duncanmmacleod opened the pull request:
This PR adds a (de)activate scripts to (un)set the `LALPULSAR_DATADIR` environment variable. This is basically the only useful part of `lalpulsar-user-env.sh` when working in a conda environment.

cc @skymoo

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

duncanmmacleod committed:
added activate/deactivate scripts

to set LALPULSAR_DATADIR in the containing environment, also added `error_overlinking` build flag

duncanmmacleod committed:
MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.5.0, and conda-forge-pinning 2019.10.01

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * The build section contained an unexpected subsection name. error_overlinking is not a valid subsection name.

duncanmmacleod committed:
add `build/error_overdepending` flag

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * The build section contained an unexpected subsection name. error_overdepending is not a valid subsection name.
 * The build section contained an unexpected subsection name. error_overlinking is not a valid subsection name.

duncanmmacleod committed:
remove unnecessary libblas host requirement

MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.5.0, and conda-forge-pinning 2019.10.01

duncanmmacleod committed:
remove libblas dependency

MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.5.0, and conda-forge-pinning 2019.10.01

duncanmmacleod committed:
added missing linux (overlinked) dependencies
